src/hdx_cli_toolkit/ckan_utilities.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import json
+import urllib3

```

Comment:
This plus `ckanapi` should be added to the requirements

Reply:
I've added it for explicitness but I believe it already imported by `hdx-python-api`